templates/provider/cluster-api/Chart.yaml

```diff
@@ -21,5 +21,6 @@ version: 0.2.2
 appVersion: "1.9.7"
 annotations:
   cluster.x-k8s.io/v1beta1: ""
+  cluster.x-k8s.io/v1alpha2: ""
```

Comment:
what is this and why? there is no such version

I would even say, that the CAPI no more has the alpha versions